src/sidebar/services/annotations-exporter.tsx

```diff
@@ -152,6 +152,89 @@ export class AnnotationsExporter {
     return `${headers}\n${annotationsContent}`;
   }
 
+  buildHTMLExportContent(
+    annotations: APIAnnotationData[],
+    {
+      groupName = '',
+      displayNamesEnabled = false,
+      defaultAuthority = '',
+      /* istanbul ignore next - test seam */
+      now = new Date(),
+    }: ExportOptions = {},
+  ): string {
+    const { uri, title, uniqueUsers, replies, extractUsername } =
+      this._exportCommon(annotations, {
+        displayNamesEnabled,
+        defaultAuthority,
+      });
+
+    return renderToString(
+      <html lang="en">
+        <head>
+          <title>Annotations export - Hypothesis</title>
+          <meta charSet="UTF-8" />
+        </head>
+        <body>
+          <section>
+            <h1>Summary</h1>
+            <p>
+              <time dateTime={now.toISOString()}>{formatDateTime(now)}</time>
```

Comment:
I don't think we should tackle this in this PR, but I noticed when testing this in VoiceOver that it would read a date such as "2023-12-01 14:05" as "two thousand twenty-three to two-thousand twelve", which is wrong.

Reply:
Hmm. I would have expected the use of `time` with the `timestamp` attribute to be used by screen readers to know that's a date, but clearly something is missing.

I'll give it a thought.